### DISCUSSION TITLE
Facing problem using allure reporter for flutter automation with testNG

### DISCUSSION THREAD

**Discussion by shoumik121** [appium/appium#20147] [General]
Is your feature request related to a problem? Please describe.

I have created appium-flutter-finder JAR file from -->https://github.com/appium/appium-flutter-driver/tree/main/finder/kotlin
then imported it in my project as JAR file dependency

When i select --> run as testNG the automation works

but i can not use the command--> ./mvnw verify
(as it is told in this doc https://allurereport.org/docs/testng/)

I get this
ERROR MSG-->

TalentPro@DESKTOP-DO44MD4 MINGW64 ~/Documents/fanfare-allure-reporter
$ ./mvnw verify
[INFO] Scanning for projects...
[INFO]
[INFO] --------------< org.example.testng:allure-testng-example >--------------
[INFO] Building allure-testng-example 1.0-SNAPSHOT
[INFO] from pom.xml
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for pro.truongsinh:appium-flutter-finder:jar:0.0.6 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 0.630 s
[INFO] Finished at: 2024-05-23T16:47:52+06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project allure-testng-example: Could not resolve dependencies for project org.example.testng:allure-testng-example:jar:1.0-SNAPSHOT: The following artifacts could not be resolved: pro.truongsinh
:appium-flutter-finder🫙0.0.6 (absent): pro.truongsinh:appium-flutter-finder🫙0.0.6 was not found in https://repo.maven.apache.org/maven2 during a previous attempt. This failure was cached in the local repository and resolution is not reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException

Describe the solution you'd like
please help me generate a testing report for my java flutter automation
I am very new to this.

Describe alternatives you've considered
i need a good test reporter any

Additional context
I am using jdk 17 if it is of any use

**Comment by sirkev** [comment]
Hello, running into the same issue here, I'm considering using browserstack to get the reports, but I would prefer to stick with allure, did you get any alternitives on your side?
using allure, I keep getting "ALLURE REPORT UNKNOWN
unknown - unknown (Unknown)"
